### ISSUE TITLE
Support for C#9 

### ISSUE THREAD

**Issue by karelkral** [RafaelSalguero/CSharp2TS#18]
Hello, the current version does not support C#9 features bellow:

* C# 9 init modifier. Conversion of this class fails.
```
        public class PoirotAutoRunModeDto
	{
		public int PoirotAutoRunMode { get; init; }
		public string Description { get; init; }
	}
```

* C# 9 records. Conversion of this class fails.
```
        public record PoirotAutoRunModeDto
	{
		public int PoirotAutoRunMode { get; init; }
		public string Description { get; init; }
	}
```

* C# 9 records. Conversion of this class fails.
```
        public record PoirotAutoRunModeDto(
              public int PoirotAutoRunMode,
              public string Description
        );
```

**Comment by RafaelSalguero** [comment]
Fixed in #5855e06cf7288e68474a67d

**Comment by karelkral** [comment]
Many thanks for the quick fix!
